src/pages/ItemDetail/ItemDetail.js

```diff
@@ -1,8 +1,157 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './ItemDetail.scss';
 
 const ItemDetail = () => {
-  return <div />;
+  const [data, setData] = useState([]);
```

Comment:
어떤 데이터인지 알 수 있게 state 명칭 명확하게 써주세요!
그리고 해당 데이터는 배열이 아닌 객체이기 때문에 초기값으로 빈 객체를 넣어주시면 될 것 같습니다.
```suggestion
  const [data, setData] = useState({});
```